### PR TITLE
Rename submit() to something that reflects the verb of the builder

### LINKED ISSUE #2029: Rename submit() to get() in Nessie API helpers
As discussed in #2011 the `.submit()` call that makes API parameter builders execute actually API requests may be confusing as it may be interpreted as providing asynchronous behaviour.

Suggest renaming it to `get()` or `retrieve()`.

### PR DESCRIPTION
**Pull request by nastra**
Fixes #2029 

This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/2074)

### CODE REVIEW

File: ui/package-lock.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
         "@types/react-dom": "^17.0.9",
         "@types/react-router-dom": "^5.1.9",
         "@types/yup": "^0.29.13",
+        "@typescript-eslint/eslint-plugin": "^4.31.2",
```
Comment (nastra) [inline ui/package-lock.json:31]: I always get this change when running a `./mvnw clean install -DskipTests=true`, so I figured it's finally worth committing it

Comment (dimas-b) [reply ui/package-lock.json:31]: I never got that change, but it look innocent :)

**Review by dimas-b — APPROVED** [review]

**Comment by dimas-b** [comment]
Do you want to reference/close #2029 with this PR?

**Comment by dimas-b** [comment]
Rebasing might help with the coverage error... IIRC it help is some of my PRs.

File: ui/package-lock.json (hunk 1 above)
Comment (dimas-b) [reply ui/package-lock.json:31]: I actually see this change with more recent branches too. So +1 to commit it.

**Comment by nastra** [comment]
> Do you want to reference/close #2029 with this PR?

yes

**Review by snazy — APPROVED** [review]

**Pull request merged**